src/utils/commandUtils.ts

```diff
@@ -36,6 +36,11 @@ export async function searchTestLocation(fullName: string): Promise<ILocation[]>
         JavaTestRunnerDelegateCommands.SEARCH_TEST_LOCATION, fullName) || [];
 }
 
+export async function resolveSourceUri(trace: string, projectNames: string[]): Promise<string> {
```

Comment:
why not change the name to resolveStackTraceLocation directly?

Reply:
ok